File: C_Programming/Practice/program15.c
#include<stdio.h>

#define DIVISOR 5

void CheckDivisibleBy5(int Number)
{
    if(Number % DIVISOR == 0)
    {
        printf("%d is Divisible by \n", Number, DIVISOR);
    }
    else
    {
        printf("%d is Not Divisible by \n", Number, DIVISOR);
    }
}

int main()
{
    int Number = 0;

    printf("Enter Number : ");
    scanf("%d", &Number);

    CheckDivisibleBy5(Number);

    return 0;
}

File: Java/Practice/program15.java
import java.util.Scanner;
public class program15 
{
    public static boolean CheckEvenOdd(int iNo)
    {
        if((iNo % 2) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
    public static void main(String[] args) 
    {
        int iValue = 0;
        boolean bRet = false;

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter number to check whether it is Even or Odd : ");            
        iValue = sc.nextInt();

        if(bRet == true)
        {
            System.out.println(iValue + " is Even");
        }
        else
        {
            System.out.println(iValue + " is Odd");
        }

        sc.close();
    }
}

/*

#include<stdio.h>
#include<stdbool.h>

bool CheckEvenOdd(int iNo)
{
    if((iNo % 2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Enter number to check whether it is Even or Odd : ");
    scanf("%d",&iValue);

    bRet = CheckEvenOdd(iValue);

    if(bRet == true)
    {
        printf("%d is Even\n",iValue);
    }
    else
    {
        printf("%d is Odd\n",iValue);
    }

    return 0;
}

*/
